scripts/llm/ptq.py

```diff
@@ -69,7 +69,7 @@ def get_args():
         "--algorithm",
         type=str,
         default="fp8",
-        choices=["no_quant", "int8", "int8_sq", "fp8", "int4_awq", "w4a8_awq", "int4", "nvfp4"],
+        choices=["no_quant", "int8", "int8_sq", "fp8", "block_fp8", "int4_awq", "w4a8_awq", "int4", "nvfp4"],
```

Comment:
NIT: this could be `choices=QUANT_CFG_CHOICES`. We'll change that later (bearing in mind possible import issues from the other file)

Reply:
Note: I have changed this in https://github.com/NVIDIA/NeMo/pull/12709 in order to make `QUANT_CFG_CHOICES` functional across different modelopt versions (this is somehow related to "import guarding")